CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## master (unreleased)
 
+### New features
+
+* [#8](https://github.com/rubocop-hq/rubocop-minitest/pull/8): Add new `Minitest/AssertTruthy` cop. ([@abhaynikam ][])
+
+[@abhaynikam]: https://github.com/abhaynikam
```

Comment:
Please move this link to the bottom of the changelog.

Reply:
Done. Please have a look.